app/src/main/java/com/example/cryptocurency/ui/add_asset/AddAssetActivity.kt

```diff
@@ -0,0 +1,114 @@
+package com.example.cryptocurency.ui.add_asset
+
+import android.os.Handler
+import android.os.Looper
+import android.view.View
+import com.example.cryptocurency.data.models.Asset
+import com.example.cryptocurency.data.models.Coin
+import com.example.cryptocurency.databinding.ActivityAddAssetBinding
+import com.example.cryptocurency.utils.*
+import com.example.cryptocurency.utils.base.BaseActivity
+import com.example.cryptocurency.utils.extension.showToast
+import com.example.cryptocurency.utils.factory.PresenterFactory
+import com.example.cryptocurency.utils.factory.SpinnerFactory
+
+class AddAssetActivity : BaseActivity<ActivityAddAssetBinding>(ActivityAddAssetBinding::inflate),
+    AddAssetActivityContract.View {
+
+    private var mPresenter: AddAssetActivityContract.Presenter? = null
+    private var coinData = mutableListOf<Coin>()
```

Comment:
```suggestion
    private var coinsData = mutableListOf<Coin>()
```
biến có kiểu danh sách thì nên khai báo là số nhiều